third_party/blink/web_tests/external/wpt/webnn/conformance_tests/optimize_graph.https.any.js

```diff
@@ -128,6 +128,105 @@ const optimizedgraphTests = [
       }
     }
   },
+  {
+    'name': 'Lesser + Where & output: [Less] -> Bool -> [Where] & [Less] -> [Cast] -> uint8(output)',
```

Comment:
Keep the name simple, add a comment that illustrating the topology
```suggestion
    'name': 'Lesser output is consumed by Where and used as graph output',
```

Reply:
Done.